### ISSUE TITLE
⌨ Mobile menu link targets too small

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#13237]
## ⌨ Accessibility Issue

When using the desktop menu, I frequently miss links in the dropdowns due to the small targets given.

### WCAG 2.1 Criteria

- SC SC 2.5.8 - Target Size (Minimum)
  Ensure targets meet a minimum size or have sufficient spacing around them.
- SC 2.5.5 – Target Size
  Interactive elements (such as buttons) should have a target size of at least 44×44 CSS pixels unless they meet an exception.

This technically meets the AA requirement of 24 exaclty but I still struggle with it so it could be helpful to increase it to bring it closer to the "enhanced" (AAA) 2.5.5 criteria.

## 🦋 Expected Behaviour

Menu items have sufficient target size so that they are easy to activate.

## 📋 Steps to Reproduce

1. Open a menu dropdown
2. Obersve the hitbox is just the next with nothing else
3. See that clicking near the text does not work, you must be right on top

## 📸 Screenshot

![Image](https://github.com/user-attachments/assets/06fbdf58-0ade-49f8-ae72-759e00e8d689)

## 🙋‍♀️ Proposed Solution

Increase the hit state to extend the entire width of the dropdown and has more vertical space.

## ✅ Acceptance Criteria

- [ ] Dropdown menu links have increased hitbox
